components/searchwithin/metadata/searchwithin.yml

```diff
@@ -11,6 +11,12 @@ sections:
 
       Since Picker's markup has changed, `.spectrum-SearchWithin-pickerTrigger` is no longer required.
 
+      ### `.spectrum-SearchWithin-clearButton` must be added to `.spectrum-ClearButton`
+      The `.spectrum-SearchWithin-clearButton` class is now required on the `.spectrum-ClearButton`.
```

Comment:
Interestingly, the markup was always wrong on this page and it never used the `.spectrum-SearchWithin-clearButton` class, it used `.spectrum-Search-clearButton`, which looked very similar. Since we're effectively changing the markup (despite having not changed any CSS), this is a breaking change.

Reply:
@GarthDB if you like, I can remove this commit from this PR since we'll be making another breaking change to get searchwithin up to Express standards. Thoughts?